src/CodeGen_LLVM.cpp

```diff
@@ -3660,11 +3660,13 @@ void CodeGen_LLVM::return_with_error_code(llvm::Value *error_code) {
 }
 
 void CodeGen_LLVM::visit(const ProducerConsumer *op) {
+    producer_consumer_id++;
+
     string name;
     if (op->is_producer) {
```

Comment:
Why the underscore in the name, given that the rest of the name is space-separated?

Reply:
I think the pattern Darya went for was `<id>_produce|consume <varname>`. I can change it to `<id>_produce|consume_<varname>`. As long as its a uniquely identifying string, it should work for our usecase.